test/workbox-routing/node/route.mjs

```diff
@@ -0,0 +1,129 @@
+import {expect} from 'chai';
+
+import expectError from '../../../infra/utils/expectError.js';
+
+(async () => {
```

Comment:
why are you wrapping everything in an iife like this?

Reply:
I'll see if the changes we discussed on Slack will allow the dynamic `async import()` to work.

Reply:
I am still having trouble without the `iife`, but things work with the top-level `async`, which is kind of standard for using `await`. I'm inclined to leave as-is and investigate further outside of this PR when I work on my next set of tests.